### PR TITLE
Mobile - Fix Experiments native variant breakage

### PR DESCRIPTION
**Pull request by geriux**
## What?
This PR brings changes from https://github.com/WordPress/gutenberg/pull/47229 to the `experiments` native variant.

## Why?
This change was breaking the editor.

## How?
By bringing the latest changes from the `experiments` file.

## Testing Instructions
CI checks should pass.

### Testing Instructions for Keyboard
N/A

## Screenshots or screencast 
N/A

### CODE REVIEW

**Comment by fluiddot** [comment]
@geriux looks like the iOS E2E tests failed 😭, not sure if was caused due to flakiness or not. I checked the video captured during the test and noticed that the editor froze when rendering the content until the test timed out. I've seen this issue on past failed test runs but I don't know what's the culprit.

Screenshot 2023-02-06 at 18 54 02

**Comment by geriux** [comment]
> @geriux looks like the iOS E2E tests failed 😭, not sure if was caused due to flakiness or not. I checked the video captured during the test and noticed that the editor froze when rendering the content until the test timed out. I've seen this issue on past failed test runs but I don't know what's the culprit.

Interesting! I'll check it out.

**Comment by geriux** [comment]
Hey @fluiddot 👋 I'm working on another PR to fix the mobile E2E tests, It'd be awesome if we can merge this one that actually breaks the editor, so I can work on the other one separately. Would you mind reviewing this so we can merge it? Even if the iOS test fails because it gets frozen before finishing the test I think we should merge this one. What do you think?

**Comment by geriux** [comment]
After a re-run the iOS test passed so we could merge this if it gets approved.

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

As referenced in [this comment](https://github.com/WordPress/gutenberg/pull/47792#issuecomment-1420634394), CI jobs succeeded.

**Review by adamziel — COMMENTED** [review]
LGTM ❤️  Thank you for the fix!

**Comment by ntsekouras** [comment]
Cherry-picked this PR to the wp/6.2 branch.

**Pull request merged**
